### PR TITLE
fix: token border colors

### PR DESCRIPTION
**Pull request by fbwoolf**
Fixes/updates some changes needed to remove current hardcoded values in the extension.

### CODE REVIEW

File: packages/tokens/src/colors.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const colors = {
+  current: { value: 'currentColor' },
+  overlay: { value: 'rgba(0,0,0,0.4)' },
```
Comment (fbwoolf) [inline packages/tokens/src/colors.ts:3]: I accidentally removed these in my previous update.

File: packages/tokens/src/tokens.ts [hunk 2]
```diff
@@ -1,8 +1,27 @@
+import { colors } from './colors';
+
 export const tokens = {
   animations: {
-    spin: {
-      value: 'spin 1s linear infinite',
-    },
+    spin: { value: 'spin 1s linear infinite' },
+  },
+  borders: {
```
Comment (fbwoolf) [inline packages/tokens/src/tokens.ts:7]: Some reorg in this file, but mostly catching some token color updates that were broken for borders with the change from `accent` to `ink`.

File: packages/tokens/src/colors.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export const colors = {
+  current: { value: 'currentColor' },
```
Comment (kyranjamie) [inline packages/tokens/src/colors.ts:2]: Saw this before but not a fan tbh. Why not just use currentColor?

**Review by kyranjamie — APPROVED** [review]

**Review by pete-watters — APPROVED** [review]

File: packages/tokens/src/colors.ts (hunk 3 above)
Comment (pete-watters) [reply packages/tokens/src/colors.ts:2]: I tracked this back to a PR I did [here](https://github.com/leather-wallet/extension/pull/4602/files) and it was already set as `current: { value: 'currentColor' }` then, I'm not sure why it's like that. 

I think we can remove it if `currentColor` will still work as that is used quite a lot in SVGs.

**Pull request merged**